### PR TITLE
DDF-05052 Added disable CN config to CSW Federated

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Added disable CN config to CSW Federated
#### Select relevant component teams: 

@Schachte @hayleynorton 
#### Ask 2 committers to review/merge the PR and tag them here.

@adimka
@jrnorth

#### How should this be tested?

Build / Install / Configure CSW Federated Profile / Verify as working
#### What are the relevant tickets?
Fixes: #__5052__

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -17,11 +17,11 @@
     <OCD name="CSW Specification Profile Federated Source" id="Csw_Federated_Source"
          description="CSW Specification Profile Federated Source should be used when federating to an external CSW service.">
 
-        <AD description="The unique name of the Source" name="Source ID" id="id" required="true"
+        <AD description="The unique name of the Source" name="Source ID" id="id"
             type="String"/>
 
         <AD description="URL to the endpoint implementing the Catalogue Service for Web (CSW) spec"
-            name="CSW URL" id="cswUrl" required="true" type="String"
```
Comment (bdeining) [inline catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/metatype/metatype.xml]: `required="true"` is redundant

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
https://github.com/codice/ddf/issues/5052

**Comment by brianfelix** [comment]
Hero was successful. Set up DDF with a CSW Federation Profile Source and was able to successfully issue queries with CN check on and off.

**Pull request merged**